### PR TITLE
support etags-select close #910 and #683 #668 #557 #699

### LINKED ISSUE #910: support for etags-select
I recently discovered etags-select, which is the best way yet that I've seen to search etags without installing a large framework like helm.

I've found that the following is enough to get the support integrated with the tag table built by projectile, incase anybody wants to do similar:

```elisp
(defun projectile-etags-select-find-tag ()
  "Run `etags-select-find-tag' in the current projectile context."
  (interactive)
  (projectile-visit-project-tags-table)
  (etags-select-find-tag))
```

of course it would be great if this was the default behaviour for `C-c p j`

### PR DESCRIPTION
**Pull request by fommil**
Could somebody who uses ggtags please test this? I'm guessing the behaviour there.

### CODE REVIEW

**Comment by bbatsov** [comment]
You'll have to rebase this.

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 * Consider `TAGS` and `GTAGS` root markers.
 * Add relation between the `.h`, `.cxx`, `.ixx` and `.hxx` files in `projectile-other-file-alist`.
 * Add support to specify project name either via `.dir-locals.el` or by providing a customized `projectile-project-name-function'.
+* Use external find-tag implementations.
```
Comment (bbatsov) [inline CHANGELOG.md]: To some extent we've done this in the past well, so here you should probably say "always use"...

This should be in changes, not new features. Guess you can add two entries - one about the etags-select here and one about dropping projectile's tag parsing in changes.

**Comment by fommil** [comment]
rebasing...

**Comment by fommil** [comment]
Thanks! Can you please close all the linked tickets?

**Pull request merged**